web/test/pages/loading.test.tsx

```diff
@@ -53,6 +55,10 @@ describe("loading page", () => {
     useMockRouter({});
   });
 
+  afterEach(() => {
+    process.env.FEATURE_LOGIN_PAGE = "true";
```

Comment:
pebble - When dealing with environment variables in test files, it’s essential to ensure their values are reverted to their original state after each test. Otherwise, the environment variable changes could persist across tests, leading to unexpected behaviors and flaky tests.

Reply:
Good callout. This line _is_ reverting it to the original state. [I added a test](https://github.com/newjersey/navigator.business.nj.gov/pull/9418/files/246ca6975ea92409b2845a77f0c4998a25802cc4#diff-c9b586be7002a1d5f2f315cb3555672e97bf490411d5ea1660443710151ebe9aR137-R144) for different expected behavior in the `false` case of the feature flag and needed to clean up afterwards, threw it in an `afterEach` because that seemed more readable for the full test file.